nixos/tests/kafka.nix

```diff
@@ -74,6 +74,10 @@ let
   }) { inherit system; });
 
 in with pkgs; {
-  kafka_2_7  = makeKafkaTest "kafka_2_7"  apacheKafka_2_7;
   kafka_2_8  = makeKafkaTest "kafka_2_8"  apacheKafka_2_8;
+  kafka_3_0  = makeKafkaTest "kafka_3_0"  apacheKafka_3_0;
+  kafka_3_1  = makeKafkaTest "kafka_3_1"  apacheKafka_3_1;
+  kafka_3_2  = makeKafkaTest "kafka_3_2"  apacheKafka_3_2;
```

Comment:
Do we really need to keep all those versions?

Reply:
Yes, I think it makes sense, 2.8 is the newest in 22.05, and then we can remove everything up to but excluding the newest version in the next NixOS release.

Reply:
I guess you need them when upgrading?

Reply:
Yes, being able to roll back and forward within a NixOS version is very useful. I think it would be a good policy to simply, for a given $release, always keep whatever version was the newest in $oldrelease and drop all older versions.